CONTRIBUTING.md

```diff
@@ -1,8 +1,16 @@
 # Contributing to ClickHouse
 
+ClickHouse is an open project, and you can contribute to it in many ways. You can help with ideas, code, or documentation. We appreciate any efforts that help us to make the project better.
+
+Thank you :-)
```

Comment:
smiles are inappropriate style for documentation itself and I think we'd better avoid it on such a visible place